Sources/Styleguide/PointFreeBaseStyles.swift

```diff
@@ -40,6 +46,24 @@ extension Class {
       public static let footnote = CssSelector.class("pf-footnote")
       public static let callout = CssSelector.class("pf-callout")
     }
+
+    public enum components {
+      /// The standard nav bar style.
+      public static let navBar =
```

Comment:
this is cool! we have a single `navBar` selector that can be added to a dom element, i.e. `div([`class`([Class.pf.components.navBar])])`, and since `navBar` is just a union of a whole bunch of other selectors, all of those names get rendered into the class attribute, i.e.`class="nav-bar bg-purple px-2 lh-4 h-4"`, etc.

makes it nice to build components from the single styles!